proxy/http/HttpSM.cc

```diff
@@ -191,6 +191,26 @@ HttpVCTable::remove_entry(HttpVCTableEntry *e)
     free_MIOBuffer(e->write_buffer);
     e->write_buffer = nullptr;
   }
+  if (e->read_vio != nullptr && e->read_vio->cont == sm) {
+    // Cleanup dangling i/o
+    if (e == sm->get_ua_entry() && sm->get_ua_txn()) {
```

Comment:
style consistency - if you check vs. `nullptr` in the previous `if`, do that here too with `sm->get_ua_txn()`.